README.md

```diff
@@ -37,11 +37,11 @@ In witness automata, the GraphML nodes represent states.
 
 | key | Meaning | Allowed in Violation Witnesses | Allowed in Correctness Witnesses |
 | --- | --- | ---- | ---- |
-| entry | *Valid values:* ``false`` (default) or ``true`` <br /> This state represents the initial state of the automata (entry state of the graph). Only exactly one initial state (entry state) is allowed. | Yes | Yes |
-| sink | *Valid values:* ``false`` (default) or ``true`` <br />  This state is a sink. All paths that lead to this state end here and should not be further explored by the witness validator. Nodes where this flag is set must not have any leaving transitions | Yes | No |
-| violation | *Valid values:* ``false`` (default) or ``true`` <br /> The witness claims that paths that reach this state violate the specification. A violation witness is only accepted if the witness validator detects a specification violation and the witness automaton is in a state where this flag is set. | Yes | No |
-| invariant | *Valid values:* A C expression that must evaluate to the C type ``int`` (used as boolean) and **may** consist of conjunctions or disjunctions, but not function calls. Local variables that have the same name as global variables or local variables of other functions can be qualified by using a data tag with the ``invariant.scope`` key. | No | Yes |
-| invariant.scope | *Valid values:* Function name <br /> The witness validator must map the variables in the given invariants to the variables in the C code. Due to scopes in C, there may be name conflicts. The witness validator will first look for a variable with a matching name in the scope of the provided function name before checking the global scope. This tag applies to the invariant as a whole. It is not possible to specify invariants about local variables of different functions. There is currently no support for different variables with the same name within different scopes of the same function. | No | Yes |
+| entry | *Valid values:* ``false`` (default) or ``true`` <br /> ``entry`` is used to mark a ``node`` as an entry node. An entry node represents the initial control state of the witness automaton. We require that exactly one initial control state is defined per document. The ``attr.type`` attribute of this key is ``boolean``. The default value is ``false``. | Yes | Yes |
```

Comment:
what is an "intitial control state"? how does it differ from "initial state"? 

> The default value is ``false``

Is this supposed to mean that I cannot set a different default value? If that is not intended, a better wording would probably be something like "if no default value is specified, it is assumed to be false"

Reply:
"initial state" is underspecified. There are many types of states. Witness automata consist of control states.

> Is this supposed to mean that I cannot set a different default value? If that is not intended, a better wording would probably be something like "if no default value is specified, it is assumed to be false"

Good point. Your suggested wording might be confusing in the paper without explaining that graphml allows specifying default values in the header, but here, it makes a lot of sense.